packages/plugin-relay/src/input-field-builder.ts

```diff
@@ -37,7 +36,7 @@ inputFieldBuilder.globalIDList = function globalIDList<Req extends FieldRequired
             (Array.isArray(forTypes) ? forTypes : [forTypes])) as ObjectRef<SchemaTypes>[]
         )?.map((type: ObjectRef<SchemaTypes>) => ({
           typename: this.builder.configStore.getTypeConfig(type).name,
-          parse: type instanceof NodeRef ? type.parseId : undefined,
+          parseId: 'parseId' in type ? type.parseId : undefined,
```

Comment:
Duck-type was the best I could come up with here, without moving `NodeRef` out of `plugin-relay` into `core` so it could be used by both plugins. Seems like it should be fine, since it shouldn't be confused with `parseId` anywhere else